docs/ErrorHandlingRationale.rst

```diff
@@ -1,2041 +0,0 @@
-Error Handling Rationale and Proposal
-*************************************
-
-.. contents::
```

Comment:
The markdown file needs a manually written index. In `.rst` files, it is rendered automatically using this directive.